events/events.go

```diff
@@ -196,3 +204,22 @@ func (e *Events) unsubscribe(l listener) bool {
 	e.mu.Unlock()
 	return false
 }
+
+func GetContainerId(eng *engine.Engine, name string) string {
+	var bytesb bytes.Buffer
+	job := eng.Job("container_inspect", name)
+
+	var outStream, errStream io.Writer
+
+	outStream = &bytesb
+	errStream = outStream
+	job.Stdout.Add(outStream)
+	job.Stderr.Set(errStream)
```

Comment:
Why use `Add` for stdout and `Set` for stderr ? Let's use `Set` for both I think